ansible/testbed-cli.sh

```
@@ -905,6 +929,8 @@ case "${subcmd}" in
                ;;
   collect-show-tech) collect_show_tech $@
                ;;
+  config-vs-chassis) config_vs_chassis $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)